validation/glean/baseline.2.core.pass.json

```diff
@@ -8,7 +8,8 @@
     "client_id": "6ff20eb7-e80d-4452-b45f-2ea7e63547aa",
     "seq": 1,
     "start_time": "2018-10-23 11:23:15-04:00",
-    "end_time": "2018-10-23 11:23:15-04:25"
+    "end_time": "2018-10-23 11:23:15-04:25",
+    "first_run_date": "2018-10-23 11:23:15-04:25"
```

Comment:
Are we doing true second resolution for `first_run_date`?

Reply:
Nope.  Will fix in the validation content.